### DISCUSSION TITLE
Dispatching events from Blade templates

### DISCUSSION THREAD

**Discussion by addtobag** [livewire/livewire#5812] [7. Bugs (archived)]
Hello, really excited to see how Livewire 3 progresses over the coming months.

I think i found an documentation error under Events and "Dispatching events from Blade templates"

The example shows:

`<button wire:click="$dispatch('show-post-modal', { id: {{ $post->id }} })">`

but for this to work i had to wrap $post->id in ''.

`<button wire:click="$dispatch('show-post-modal', { id: '{{ $post->id }}' })">`

Please correct me if i'm wrong on this, i'm using Ulid for IDs, so this may contribute to the issue i was having as well.

Thanks

Ross

**Comment by AT0047** [comment]
Hi 
I tried that too but it doesn't work :(
